AmazonChimeSDK/AmazonChimeSDK/audiovideo/audio/AudioMode.swift

```diff
@@ -19,8 +19,6 @@ import Foundation
     /// The stereo audio mode with two audio channels for speaker, and single audio channel for microphone, both with 48KHz sampling rate.
     case stereo48K = 3
 
-    case nodevice = 4
```

Comment:
Same as the comment for Android

Reply:
Left a comment about reserving 4 for `nodevice` because I couldn't find a clean way to prevent reusing this value in the future

Reply:
Updated to use the `available` attribute and added extra handling to avoid using value 4